src/wanna/cli/version.py

```diff
@@ -1,133 +1,45 @@
-import re
-from datetime import datetime, timedelta
-from html.parser import HTMLParser
-from pathlib import Path
-from typing import Any, List, Optional, Union
+import json
 
+import pkg_resources
 import requests
-import typer
-from packaging import version
 
 from wanna.core.loggers.wanna_logger import get_logger
 
-from .. import __version__
-
 logger = get_logger(__name__)
 
-ARTIFACTORY_URL = "https://artifactory.ida.avast.com/artifactory/api/pypi/pypi-local/simple/wanna-ml/"
-VERSION_CHECK_FILE = "last_version_check"
+PYPI_URL = "https://pypi.org/pypi/wanna-ml/json"
 UPDATE_MESSAGE = (
     "If you used `pipx` to install Schnitzel CLI, use the following command:\n\n"
```

Comment:
Please also change this, we dont need Schnitzel mention in the code, I am suprised it is even here :D